### ISSUE TITLE
HDF5Reader conflict with current readers

### ISSUE THREAD

**Issue by borondics** [Quasars/orange-spectroscopy-prototypes#11]
The addition of the [HDF5Reader](https://github.com/Quasars/orange-spectroscopy-prototypes/blob/23d0fe359d1a3ff0efbd7ee7df92217246b8f2e9/orangecontrib/protospec/data.py#L21
) seems to break the currently active file readers.

**Comment by stuart-cls** [comment]
Right, there's no handling of the extension conflict in this version. (I assume you mean the auto-identification is broken, not the readers themselves, since all the tests pass).

The long-term solution is to fix our low-priority readers with respect to this one, but that only really makes sense once this reader is merged into Orange (https://github.com/biolab/orange3/pull/6791).

I'll think if there's a good temporary workaround, but for now you will need to manually select the reader you would like.

Caveat I interpreted your bug report correctly, otherwise more details please!
